jung-io/src/main/java/edu/uci/ics/jung/io/PajekNetReader.java

```diff
@@ -256,7 +256,8 @@ private void readVertex(String curLine, List<V> id, int num_vertices) {
       String[] initial_split = curLine.trim().split("\"");
       // if there are any quote marks, there should be exactly 2
       Preconditions.checkArgument(
-          initial_split.length == 2, "Unbalanced (or too many) " + "quote marks in " + curLine);
+          initial_split.length == 2 || initial_split.length == 3,
+          "Unbalanced (or too many) quote marks in " + curLine);
```

Comment:
By using `"Unbalanced (or too many) quote marks in %s", curLine`, string concatenation would only happen in case of an actual error.

Reply:
Yes, that's true.  There are a number of things like this in the code that need to be cleaned up.  (In this context, we also might want to be using `Splitter`.)  We'll be doing cleanup passes for several different kinds of issues before release.

Thanks!